server/src/main/java/org/opensearch/index/translog/RemoteFsTranslog.java

```diff
@@ -75,6 +78,8 @@ public class RemoteFsTranslog extends Translog {
     private static final int REMOTE_DELETION_PERMITS = 2;
     private static final int DOWNLOAD_RETRIES = 2;
     public static final String TRANSLOG = "translog";
+    public final static String METADATA_DIR = "metadata";
+    public final static String DATA_DIR = "data";
```

Comment:
can we unify tlog and segments `data` and `metadata` prefix names ?

Reply:
Let me do this, I was thinking of creating an enum for the same.

Reply:
Addressed your comment.